app/src/main/java/com/duckduckgo/app/browser/duckplayer/DuckPlayerJSHelper.kt

```diff
@@ -168,6 +169,9 @@ class DuckPlayerJSHelper @Inject constructor(
             "openSettings" -> {
                 return OpenDuckPlayerSettings
             }
+            "openInfo" -> {
```

Comment:
Any tests for this?

Reply:
Tested in next PR, https://github.com/duckduckgo/Android/pull/4872